sdk/face/ai-vision-face-rest/vitest.config.ts

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License.
 
 import { defineConfig } from "vitest/config";
+import { relativeRecordingsPath } from "@azure-tools/test-recorder";
```

Comment:
not sure why we need to import this?

Reply:
I think it's generated code. Let me remove it.

Reply:
You should just be able to extend and merge from this file https://github.com/Azure/azure-sdk-for-js/blob/main/vitest.shared.config.ts